### ISSUE TITLE
Network tab doesn't have response body

### ISSUE THREAD

**Issue by buaashuai** [flutter/devtools#1952]
Network tab doesn't have response body
![image](https://user-images.githubusercontent.com/11747264/81630435-29e25680-9438-11ea-8401-4d154c00d308.png)

**Comment by kenzieschmoll** [comment]
Did you try expanding the sections "General", "Response Headers", or "RequestHeaders"?

**Comment by MichaelMarner** [comment]
None of the expanded headers contain either the request or response body - only the headers (like the name suggests)

Using DevTools 0.2.5

**Comment by buaashuai** [comment]
@kenzieschmoll I expand all the sections. there is no section contains response body. I only find the headers of Response and Request.

**Comment by pdmshrestha** [comment]
I tried too. There is no response body. It is not that useful when you can't inspect response body.

**Comment by 01abhishekjain** [comment]
+1
This is a must. The new (very welcome and awaited) network tab is only halfway usable until it shows the complete request and response object details.

**Comment by encubos** [comment]
It would be really great if we can see and inspect the **request body** and **response body**

**Comment by hienlh** [comment]
Waiting for new update for this feature.

**Comment by slavap** [comment]
request body and response body are both must for Network tool, please add it.

**Comment by iJimmyWei** [comment]
Guessing this is blocked by https://github.com/dart-lang/sdk/issues/42952. As soon as the timeline can give back body information we can start addressing this it seems

**Comment by rknell** [comment]
Looks like it was attached to the september 2020 release milestone and was closed yesterday - hopefully it will surface in production soon!

https://github.com/dart-lang/sdk/issues/42952

**Comment by pdmshrestha** [comment]
According to https://groups.google.com/g/flutter-announce/c/24LppkXdMtM?pli=1

> Show HTTP and HTTPs response bodies in the request inspector (requires Dart 2.10 / Flutter 1.22 or later)

But still missing though I have flutter 1.22 and dart 2.10.1
![Uploading Screenshot 2020-10-14 at 10.56.54.png…]()


`✗ flutter doctor -v
[✓] Flutter (Channel stable, 1.22.1, on Mac OS X 10.15.7 19H2, locale en-GB)
    • Flutter version 1.22.1 at /Users/padam/Development/flutter
    • Framework revision f30b7f4db9 (6 days ago), 2020-10-08 10:06:30 -0700
    • Engine revision 75bef9f6c8
    • Dart version 2.10.1`

**Comment by Nico04** [comment]
> According to https://groups.google.com/g/flutter-announce/c/24LppkXdMtM?pli=1
> 
>     Show HTTP and HTTPs response bodies in the request inspector (requires Dart 2.10 / Flutter 1.22 or later)
> 
> But still missing though I have flutter 1.22 and dart 2.10.1
> Uploading Screenshot 2020-10-14 at 10.56.54.png…
> 
> ✗ flutter doctor -v [✓] Flutter (Channel stable, 1.22.1, on Mac OS X 10.15.7 19H2, locale en-GB) • Flutter version 1.22.1 at /Users/padam/Development/flutter • Framework revision f30b7f4db9 (6 days ago), 2020-10-08 10:06:30 -0700 • Engine revision 75bef9f6c8 • Dart version 2.10.1

In this [article](https://medium.com/flutter/announcing-flutter-1-22-44f146009e5f), they specify :
> To enable this feature, make sure you’re on the Flutter dev channel via flutter channel dev and flutter channel upgrade.

**Comment by pdmshrestha** [comment]
@Nico04 
`flutter channel 
Flutter channels:
  master
* dev
  beta
  stable
flutter upgrade
Flutter is already up to date on channel dev
Flutter 1.23.0-18.0.pre • channel dev • https://github.com/flutter/flutter.git
Framework • revision 37ebe3d82a (17 hours ago) • 2020-10-13 10:52:23 -0700
Engine • revision 6634406889
Tools • Dart 2.11.0 (build 2.11.0-213.0.dev)`
I still don't have response tab

**Comment by yashwp** [comment]
```
$ flutter --version
Flutter 1.22.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision d408d302e2 (2 weeks ago) • 2020-09-29 11:49:17 -0700
Engine • revision 5babba6c4d
Tools • Dart 2.10.0
```

Same, I don't have response preview in my dart dev tool.

**Comment by kenzieschmoll** [comment]
@pdmshrestha do you see http response events in the timeline? these should be connected to the http request event by a thin guideline

@yashwp you need to be on the dev channel - according to you `flutter --version` output, you are on the stable channel

**Comment by yashwp** [comment]
@kenzieschmoll When can we expect this feature on stable channel? Any estimate timeline?

**Comment by yashwp** [comment]
@kenzieschmoll I have changed the channel to dev as well, but still can see the response.

**Comment by kenzieschmoll** [comment]
@yashwp do you see HTTP response events in the timeline? They should be linked to the corresponding request by a thin guideline.

**Comment by kenzieschmoll** [comment]
closing this as a duplicate of https://github.com/flutter/devtools/issues/2398
